spec/bourbon/css3/font_face_spec_1.rb

```diff
@@ -0,0 +1,18 @@
+require "spec_helper"
+
+describe "font-face" do
+  before(:all) do
+    ParserSupport.parse_file("css3/font-face-1")
+  end
+
+  context "called with defaults" do
+    it "outputs defaults" do
+      ruleset = 'font-family: "source-sans-pro"; ' +
+                'font-style: normal; ' +
+                'font-weight: normal; ' +
+                'src: url("/fonts/source-sans-pro/source-sans-pro-regular.woff2") format("woff2"), url("/fonts/source-sans-pro/source-sans-pro-regular.woff") format("woff"), url("/fonts/source-sans-pro/source-sans-pro-regular.ttf") format("truetype");'
```

Comment:
Line is too long. [252/80]<br>Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.